lib/God/ForkMode.js

```diff
@@ -94,7 +94,7 @@ module.exports = function ForkMode(God) {
         var cspr = spawn(command, args, {
           env      : pm2_env,
           detached : true,
-          windowsHide: true,
+          windowsHide: pm2_env.windowsHide,
```

Comment:
Maybe we should do something like 
`windowsHide: pm2_env.windowsHide || true`

Just in case variable is undefined, it's probably why tests are failing.

Reply:
`windowsHide: pm2_env.windowsHide || true` wouldn't work though because `false || true` equals `true` so it will never be `false` (which is the goal).
I did try adding the following however to ensure that windowsHide is not `undefined` but it still failed the build test in travis (but succeeded on my machine):
```javascript
var windowsHide = true;
if (pm2_env.windowsHide != undefined) {
    windowsHide = pm2_env.windowsHide
}
```

Reply:
To check undefined is not enough I think, pm2_env.windowsHide can be null, or written as a string "false".
Maybe we should check if variable is a boolean, like so : 
`typeof(pm2_env.windowHide) === "boolean"`

Reply:
Will test that now ! thanks !

Reply:
The fix worked but the tests failed for node4 although it doesn't seem related to those changes, is it possible it's a build error ?

Reply:
Yes sometimes tests failed on node4. I will rerun the job.

Reply:
Thanks !